docs/quickstart.rst

```diff
@@ -367,3 +368,39 @@ the basic application code in the previous section::
 To learn more about the various Flask-Mail settings to configure it to
 work with your particular email server configuration, please see the
 `Flask-Mail documentation <http://packages.python.org/Flask-Mail/>`_.
+
+
+Localization Configuration
+==========================
+
+Flask-Security comes with translations for several languages. Localization is implemented via
+`Flask-BabelEx <https://pythonhosted.org/Flask-BabelEx/>`_, a fork of the
+`Flask-Babel <https://pythonhosted.org/Flask-Babel/>`_ package.
+
+The following three step process will enable translations/i18n for a basic application:
+
+1. Install Flask-BabelEx::
+
+    pip install flask-babelex
+
+2. Configure Flask-Security to use the correct I18N_DOMAIN::
+
+    app.config['SECURITY_I18N_DOMAIN'] = 'flask_security'
```

Comment:
This is already a default (see https://github.com/mattupstate/flask-security/blob/develop/flask_security/core.py#L48)